packages/utilities/src/Customizer.tsx

```diff
@@ -96,6 +107,7 @@ export class Customizer extends BaseComponent<ICustomizerProps> {
   private _onCustomizationChange = () => this.forceUpdate();
 }
 
+// TODO: consider folding this back into Customizer if no longer used
 export function mergeCustomizations(props: ICustomizerProps, parentContext: ICustomizerContext): ICustomizerContext {
```

Comment:
I exported this in #6391 and folding it back in would technically be an API breaking change, but I think it's good to discourage its use. I think it's too impactful on perf and I shouldn't have used it in my original approach.